packages/react/src/PatientTimeline.test.tsx

```diff
@@ -9,6 +11,12 @@ import { PatientTimeline, PatientTimelineProps } from './PatientTimeline';
 const medplum = new MockClient();
 
 describe('PatientTimeline', () => {
+  beforeAll(() => {
+    indexStructureDefinitionBundle(readJson('fhir/r4/profiles-types.json') as Bundle);
```

Comment:
Is there a way to run this once before all the unit tests for @medplum/react? That would make it easier for new test writers rather than having to ingest these resources for spefic test suites

Reply:
✅  - updated `test.setup.ts`

There are pros/cons to doing it globally, because it's not always what you want.  In both `react` and `app`, the client has to request the schema details from the server.  I chose not to make it global in `app`, because it's important that we test the code paths that make those server requests.